src/main/java/com/softserveinc/Task03.java

```diff
@@ -4,8 +4,53 @@
 
 public class Task03 {
     public static void main(String[] args) {
+
+        int waterInMachine= amount("Write how many ml of water the coffee machine has:");
+        int milkInMachine= amount(" Write how many ml of milk the coffee machine has:");
+        int coffeeBeansInMachine= amount("Write how many grams of coffee beans the coffee machine has:");
+        int cupsOfCoffeeNeed= amount("Write how many cups of coffee you will need:");
+
+        int cupsEnoughWater= enoughForCoups(waterInMachine, 200);
+        int cupsEnoughMilk= enoughForCoups(milkInMachine, 50);
+        int cupsEnoughCoffeeBeans= enoughForCoups(coffeeBeansInMachine, 15);
+
+        int possibleAmountOfCups=cupsEnoughWater;
+
+        int[] estimatedAmountOfCups = {cupsEnoughWater, cupsEnoughMilk, cupsEnoughCoffeeBeans};
+
+        for (int i:estimatedAmountOfCups)
+        {
+            if (possibleAmountOfCups>i){
+                possibleAmountOfCups=i;
+            }
+        }
+
+        if  (possibleAmountOfCups==cupsOfCoffeeNeed){
+            System.out.println("Yes, I can make that amount of coffee");
+        }
+        if (possibleAmountOfCups<cupsOfCoffeeNeed){
+            System.out.printf("No, I can make only %d cup(s) of coffee.", possibleAmountOfCups);
+        }
+
+        if (possibleAmountOfCups>cupsOfCoffeeNeed){
+                System.out.printf("Yes, I can make that amount of coffee (and even %d more than that)",(possibleAmountOfCups-cupsOfCoffeeNeed));
+            }
+        }
+
+
+    public static int amount(String question){
         Scanner scanner = new Scanner(System.in);
-        // TODO: Write your code here
+        System.out.println(question);
```

Comment:
Good. But don't forget about code style. You can use keyboard shortcut `Ctrl-Alt-L` to format your code.